### DISCUSSION TITLE
Using table selections to create filter for other data provider

### DISCUSSION THREAD

**Discussion by jakobwarnhjelm** [Budibase/budibase#10569] [Help]
I would like to use selection in the built in table object to filter another data provider.

So basically check necessary rows in one table. And then showing the same rows in another object.

I've been trying the "Selected row" binding, but I didn't get how to feed that information as input to a data provider filter.

I'm only using SQL tables.

(what I'm really trying to do is  to implement batch edit. As in select a few rows, and set a specific column to the same value for all selected rows).

**Comment by melohagan** [comment]
Hey @jakobwarnhjelm 

To do a batch edit you will need to use Automations to access [Looping](https://docs.budibase.com/docs/looping).

The `{{ ...Selected rows }}` binding is an **Array** of Row IDs:

![Screenshot 2023-05-15 at 10 30 43](https://github.com/Budibase/budibase/assets/101575380/4ad1d756-bf65-4b34-8c8a-6254a1d23daf)

This array can be passed into an [App action](https://docs.budibase.com/docs/app-action) trigger:

![Screenshot 2023-05-15 at 10 32 59](https://github.com/Budibase/budibase/assets/101575380/b4a8e645-5968-4ec1-8afe-c2b21d79a1d7)

![Screenshot 2023-05-15 at 10 34 08](https://github.com/Budibase/budibase/assets/101575380/3d996216-64c6-47e4-812f-a5b33c23406e)

**Reply by jakobwarnhjelm** [reply]
Ok thanks, I will try it and post how it goes :)

**Reply by jakobwarnhjelm** [reply]
Could you post the last part of the screenshot of "Step 2" in the automation?
